src/module_ground_station/macetogui.cpp

```diff
@@ -74,7 +74,7 @@ void MACEtoGUI::sendVehicleHome(const int &vehicleID, const command_item::Spatia
     if(home.getPosition()->getCoordinateSystemType() == CoordinateSystemTypes::GEODETIC)
     {
         QJsonObject obj = home.toJSON(0, guiMessageString(GuiMessageTypes::VEHICLE_HOME));
-        obj["name"] = QString::fromStdString("Agent " + std::to_string(vehicleID));
+        obj["name"] = QString::fromStdString(std::to_string(vehicleID));
```

Comment:
Similar to comment above: if "name" is only acting as a vehicle ID, then just change this field name accordingly.

Reply:
Name field is being used as a proxy for agent ID right now for home positions.